datafusion/proto/tests/cases/serialize.rs

```diff
@@ -243,3 +243,41 @@ fn context_with_udf() -> SessionContext {
 
     ctx
 }
+
+#[test]
+fn test_expression_serialization_roundtrip() {
+    use datafusion_common::ScalarValue;
+    use datafusion_expr::expr::ScalarFunction;
+    use datafusion_expr::BuiltinScalarFunction;
+    use datafusion_proto::logical_plan::from_proto::parse_expr;
+    use datafusion_proto::protobuf::LogicalExprNode;
+    use strum::IntoEnumIterator;
+
+    let ctx = SessionContext::new();
+    let lit = Expr::Literal(ScalarValue::Utf8(None));
+    for builtin_fun in BuiltinScalarFunction::iter() {
+        println!("Checking function: {}", builtin_fun.name());
```

Comment:
nit: we should probably remove this debug logging?

Reply:
```suggestion
```